submissions/chabuChabu/html/contact-form.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" >
+        <title>Contact-Us</title>
+    </head>
+    <body >
+        <h1>Kwathu</h1><hr
+        <div>
+          <a href="signup.html">Sign up</a>
+          <a href="product.html">Product</a>  
+        </div>
+        <h4>Thank you for getting in touch. Please fill out your details and type your inquiry in the form below, A member of our team will get in touch.</h4>
+        <form action="#" method="post" enctype="multipart/form-data">
+          <fieldset>
+            <legend><strong>Contact form</strong></legend>
+            <label>Full Name:</label><br>
+            <input type="text" name="fullname" id="fullname" ><br><br>
+            <label>Email:</label><br>
+            <input type="email" name="email" id="email" ><br><br>
+            <label>Please type your inquiry below</label><br>
+            <textarea name="comments" id="comments" cols="60" rows="10"></textarea><br><br>
```

Comment:
Your form needs an input field for the subject of the message. Kindly update @ChabuChabu